app/controllers/TopicController.php

```diff
@@ -3,19 +3,24 @@
 namespace App\Controllers;
 
 use http\Env\Request;
-use App\Core\DB;
 
-class TopicController extends controller {
+use App\Core\DB;
 
-    public function show($id) {
-        echo "Topic show - $id";
+class TopicController extends controller
+{
+    public function show($id)
+    {
+        $topics = DB::table("topics")->where('Id','=', $id)->getAll();
```

Comment:
1. It's $topic, not $topics, getAll() - returns array, it should be single object - single topic!
2. No error checking, what if topic is null (not found)?
3. Consider using Topic::find($id);